python/across_api/auth/api.py

```diff
@@ -78,15 +83,41 @@ async def __call__(self, request: Request) -> None:
         else:
             raise AssertionError("No credentials passed.")
 
+        return {}
+
 
-security = JWTBearer(
+JWTBearerSecurity = JWTBearer(
     scheme_name="ACROSS API Authorization",
     description="Enter your access token.",
 )
-JWTBearerDep = [Depends(security)]
 
 
-@app.get("/auth/verify", dependencies=JWTBearerDep)
+async def ScopeAuthenticate(
```

Comment:
This is an authorization decision. The user is already authenticated.

```suggestion
async def ScopeAuthorize(
```